lib/type.gi

```diff
@@ -13,6 +13,24 @@
 
 #############################################################################
 ##
+#M  FiltersType( <type> )
+##
+##  Return list of filters set in the type
+##
+InstallMethod(FiltersType, "for a type", [ IsType ],
+    type -> FILTERS{ TRUES_FLAGS(type![2]) });
+
+#############################################################################
+##
+#M  FiltersObj( <object> )
+##
+##  Return list of filters set in the type
+##
+InstallMethod(FiltersObj, "for an object", [ IsObject ],
+    obj -> FiltersType(TypeObj(obj)));
+
+#############################################################################
+##
 #M  NamesFilterShort( flags, max )
```

Comment:
All in all, this PR looks fine to me. Better introspection features certainly seems like a useful addition to me, be it to debug the system, or to learn.